### DISCUSSION TITLE
Tasto con pop up

### DISCUSSION THREAD

**Discussion by mriitan** [indomus/forum#1457] [HUB: Home Assistant]
Ciao ho un pow3 comandato da un tasto, siccome ho paura che posso essere premuto accidentalmente esiste qualcosa che una volta premuto faccia comparire un qualcosa per confermare il comando? Grazie

**Comment by Larry-78** [comment]
Ciao, se installi [button-card]( https://github.com/custom-cards/button-card), potresti utilizzare l'opzione [confirmation]( https://github.com/custom-cards/button-card#confirmation). 
Io la utilizzo su un paio di entità abbinata a semplici animazioni, ci sono molte altre opzioni e c'è un ampio margine di personalizzazione.

**Reply by mriitan** [reply]
grazie mille, ho risolto con una finestra condizionale, grazie mille comunque

**Reply by Larry-78** [reply]
Ok meglio così. La mia risposta seguiva la tua richiesta di pop-up con relativa conferma. Se ti fosse bastato solo far scomparire/apparire l'entità a seconda dello stato, te l'avrei suggerito anch'io. Ad ogni modo bene che hai risolto in autonomia.
